fast_llm/data/preparator/gpt_memmap/config.py

```diff
@@ -28,6 +28,30 @@
 class SourceSchemaConfig(Config):
     pass
 
+@config_class()
+class PromptCompletionConfig(SourceSchemaConfig):
+    type: typing.ClassVar[str] = "prompt_completion"
+    prompt_column: str = Field(
+        default="prompt",
+        desc="Field of the dataset to use.",
+        hint=FieldHint.optional,
+    )
+    completion_column: str = Field(
+        default="completion",
+        desc="Field of the dataset to use.",
+        hint=FieldHint.optional,
+    )
+    delimiter: str = Field(
+        default="",
+        desc="Delimiter between prompt and completion.",
+        hint=FieldHint.optional,
+    )
+    set_loss_masking_for_prompt: bool = Field(
```

Comment:
Pretty sure we don’t need this flag and can always assume True.
Reason: for training we can decide if we want to use the loss spans or not.